tests/testing-utils/src/chain_clients.rs

```diff
@@ -422,59 +493,112 @@ pub async fn create_total_delegated_stake_operator_set(
     let service_manager_address = get_service_manager_address(http_endpoint.to_string()).await;
     let service_manager =
         MockAvsServiceManager::new(service_manager_address, default_signer.clone());
-
-    service_manager
-        .setAppointee(
+    let permission_controller_address =
+        get_permission_controller_address(http_endpoint.to_string()).await;
+    let contract_permission_controller =
+        PermissionController::new(permission_controller_address, get_provider(http_endpoint));
+    if !contract_permission_controller
+        .canCall(
+            service_manager_address,
```

Comment:
Can we extract each of these into a helper? In this case named like `ensure_can_set_avs_registrar`?

Reply:
https://github.com/Layr-Labs/eigensdk-rs/pull/420/commits/17e184f8d7b261344b1795646a1a4da2bb4c17a9